challenges/lib/src/main/java/challenges/utilities/FizzBuzzTree.java

```diff
@@ -0,0 +1,38 @@
+package challenges.utilities;
+
+import challenges.tree.BinaryTree;
+import challenges.tree.Node;
+
+import java.util.ArrayList;
+
+public class FizzBuzzTree {
+
+
+    public FizzBuzzTree (){}
+
+
+
+
+    public static BinaryTree fizzBuzzTree (BinaryTree input) throws Exception {
+        if (input.root == null) throw new Exception("Oh snap, there's nothing to traverse");
+
+        helper(input.root);
+
+ return input;
+
+    }
+    private static void helper(Node inputNode){

```

Comment:
Great work with your helper method! Looks like you really got a handle on recursion!